packages/flutter/lib/src/animation/tween.dart

```diff
@@ -108,9 +119,11 @@ class IntTween extends Tween<int> {
   int lerp(double t) => (begin + (end - begin) * t).round();
 }
 
+/// Transforms the value of the given animation by the given curve.
 class CurveTween extends Animatable<double> {
```

Comment:
why is this one "CurveTween" and the other one "Curve*d*Animation"? Unless we have a good reason, we should make it more consistent.